### PR TITLE
KIALI-802 - Show error traffic in service graph traffic animation

### PR DESCRIPTION
**Pull request by josejulio**
Error percent 100%

peek 2018-05-25 15-20

Error percent 25%
peek 2018-05-25 15-43

Add red diamond shape for errors, the rate of diamonds to show is given by the `percentErr`, if none is found, the rate is 0 (no errors are show).

Also did a some tweaks on the params to show "TrafficPoints" more often.

### CODE REVIEW

File: src/components/CytoscapeGraph/graphs/TrafficRenderer.ts [hunk 1]
```diff
@@ -84,6 +100,23 @@ class TrafficPointGenerator {
...
+
+  setErrorRate(errorRate: number) {
+    this.errorRate = errorRate;
+  }
+
+  private nextPoint(): TrafficPoint {
+    console.log('errorRate', this.errorRate);
+    const isErrorPoint = Math.random() <= this.errorRate;
+    console.log('isErrorPoint', isErrorPoint);
+    console.log('speed', this.speed);
```
Comment (jmazzitelli) [inline src/components/CytoscapeGraph/graphs/TrafficRenderer.ts]: people have been complaining lately about the amount of logs. So either remove these or change to debug logs

Comment (josejulio) [reply src/components/CytoscapeGraph/graphs/TrafficRenderer.ts]: forgot to remove, thanks

**Comment by josejulio** [comment]
> is it just me seeing this or the red dots "travel" backwards

I suspect is just an optical illusion caused by the low framerate of the gifs. If you focus, you can control the direction (try blinking) :)

**Comment by josejulio** [comment]
peek 2018-05-28 20-33

**Pull request merged**